.github/workflows/build.yml

```diff
@@ -119,11 +119,11 @@ jobs:
           coveralls --service=github
 
   build-and-test-mac-gui:
-    runs-on: macos-latest
+    runs-on: macos-13
```

Comment:
We should add a comment indicating why this is locked to macOS 13.

Reply:
It's in the commit message ("fix macOS version to 13 (latest version, 14, does not have tk)"), but can add as a comment. In general though it is better to have a fixed version to improve consistency.